### ISSUE TITLE
What is the computation complexity of PointMLP-elite?

### ISSUE THREAD

**Issue by zhouchch3** [ma-xu/pointMLP-pytorch#21]
Hi and Thanks for your great work very much.
I want to deploy the elite version on mobile devices. I have two  questions:
1. What is the computation complexity of PointMLP-elite? (FLOPS)
2. How much accuracy loss if halfing the size of PointMLP-elite? (I want a more lightweight  version)
Thank you very much!

**Comment by ma-xu** [comment]
@zhouchch3 Thanks for your interest.


I uploaded a script (analysis.py) to analysis the computtional complexity, you can run it by "python analysis.py".
The results are:
Flops : 911683072
Flops : 0.8G
Params : 716232
Params : 0.7M
Note that 1) the FLOPs includes all computations from FPS and KNN, etc. 2) You need to comment Line 9, 161, and un-comment Line 160 (to remove the CUDA lib).

  
I didn't test that. But you can easily  try it by adjusting the hyperparameters in the configure (Line 356). For example, the number of blocks, the number of channels, res_expansion, etc. 

Let me know if you have further questions or concerns.

**Comment by zhouchch3** [comment]
> 
Thanks  for your detailed solutions for these two questions. 👍
I'll try your solutions. 😄
